charts/gardener-dashboard/values.yaml

```diff
@@ -204,6 +204,10 @@ global:
       features:
         terminalEnabled: false
         projectTerminalShortcutsEnabled: false
+
+      shootAdminKubeconfig:
+        enabled: true
```

Comment:
I don't think it should be enabled by default. What do you think @petersutter?

Reply:
Changed it